src/inset_state/simplify_inset.cpp

```diff
@@ -22,16 +22,10 @@ void InsetState::simplify(const unsigned int target_points_per_inset)
   for (const auto &gd : geo_divs_) {
     for (const auto &pwh : gd.polygons_with_holes()) {
       pgn_id_to_constraint_id[pgn_id++] =
+        // NOTE: This line causes a segmentation fault in
+        // cartogram ../../sample_data/world_by_country_since_2022/world_by_country_since_2022.geojson ../../sample_data/world_by_country_since_2022/world_population_2010.csv -QST
```

Comment:
This comment can be removed as this issue has been resolved.

Reply:
Actually, this still happens in debug mode. I have no idea why. I think it is still worth looking into.